### ISSUE TITLE
编译时报错

### ISSUE THREAD

**Issue by liuxuwei** [anzaizai/EasySwipeMenuLayout#51]
Could not find EasySwipeMenuLayout.aar (com.github.anzaizai:EasySwipeMenuLayout:1.1.2).
Searched in the following locations:
    https://jitpack.io/com/github/anzaizai/EasySwipeMenuLayout/1.1.2/EasySwipeMenuLayout-1.1.2.aar

貌似.aar文件找不到

**Comment by zhlofworks** [comment]
Gradle Plugin Version升级到最新：3.4.1
Gradle Version升级到5.1.1，之后遇到上面同样的问题，之前低版本没这样的报错

**Comment by zhlofworks** [comment]
@liuxuwei 使用最新版本1.1.3，应该就不会报这个错了

**Comment by ryancfogarty** [comment]
@zhlofworks 

I am having the same issue. Have also tried com.github.anzaizai:EasySwipeMenuLayout:1.1.3 and issue is still occurring.

However, com.github.anzaizai:EasySwipeMenuLayout:1.1.1 builds fine.

**Comment by monakatyking** [comment]
同问，Failed to resolvecom.github.anzaizai:EasySwipeMenuLayout:1.1.2怎么解决，换成1.1.3也不行，只有1.1.1可以

**Comment by LiHexiang** [comment]
真神奇，我昨天也突然遇到这个问题

**Comment by ryancfogarty** [comment]
https://jitpack.io/#anzaizai/EasySwipeMenuLayout

1.1.2 and 1.1.3 are reporting errors on Jitpack 

```
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugApkCopy'.
   > Could not find com.android.support.constraint:constraint-layout:1.0.0-alpha9.
     Searched in the following locations:
         file:/opt/android-sdk-linux/extras/android/m2repository/com/android/support/constraint/constraint-layout/1.0.0-alpha9/constraint-layout-1.0.0-alpha9.pom
         file:/opt/android-sdk-linux/extras/android/m2repository/com/android/support/constraint/constraint-layout/1.0.0-alpha9/constraint-layout-1.0.0-alpha9.jar
         file:/home/jitpack/build/sdk-manager/com/android/support/constraint/constraint-layout/1.0.0-alpha9/constraint-layout-1.0.0-alpha9.jar
     Required by:
         project :app
```

Looks like perhaps an outdated version of `constraint-layout`?

**Comment by zhlofworks** [comment]
@rfogar2 I think if update gradle version to latest,then constraint-layout will  not working。after use constraint-layout  it will report error when complie your app, haha,it is my guess, not sure

**Comment by elmliu** [comment]
更新了AS和Gradle版本，突然也出现了这个问题，换到1.1.1版本确实可以解决

**Comment by liuxuwei** [comment]
同样换到latest:release，  AS提示最新版本为1.1.1

**Comment by liuxuwei** [comment]
> @zhlofworks
> 
> I am having the same issue. Have also tried com.github.anzaizai:EasySwipeMenuLayout:1.1.3 and issue is still occurring.
> 
> However, com.github.anzaizai:EasySwipeMenuLayout:1.1.1 builds fine.

1.1.1 works well too

**Comment by MakeCodeFly** [comment]
我也遇到这个问题   大家怎么解决的

**Comment by tangjiaqq** [comment]
要么自己编译aar 要么 自己clone 这个库 发依赖  或者  使用1.1.1

**Comment by lucklygaj** [comment]
感谢大家的支持，last version is 1.1.4 has fix #51
